providers/aws/appsync.go

```diff
@@ -0,0 +1,50 @@
+package aws
+
+import (
+	"context"
+	"log"
+
+	"github.com/GoogleCloudPlatform/terraformer/terraform_utils"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/appsync"
+)
+
+type AppSyncGenerator struct {
+	AWSService
+}
+
+func (g *AppSyncGenerator) InitResources() error {
+	config, e := g.generateConfig()
+	if e != nil {
+		return e
+	}
+
+	svc := appsync.New(config)
+
+	// TODO:
+	// * Service does not provides a `NewXXXXXXXXXXXPaginator` like other services' "NewListClustersPaginator" right now
+	// * Replace when it being avaialble, using the max under constraint currently
+	const MaxGraphqlApis = 25
```

Comment:
In Terraformer we usually don’t provide page size when crafting requests for listing with a hope max size is default size. I’m fine with the code but I think we’d need to verify the claim and perhaps write own iterator if one is not provided by the SDK.

Reply:
❯ ag 'MaxResults \*int'  | grep 'required'
```
aws-sdk-go-v2/service/cognitoidentity/api_op_ListIdentityPools.go:19:	MaxResults *int64 `min:"1" type:"integer" required:"true"`
aws-sdk-go-v2/service/cognitoidentity/api_op_ListIdentities.go:29:	MaxResults *int64 `min:"1" type:"integer" required:"true"`
aws-sdk-go-v2/service/cognitoidentityprovider/api_op_ListUserPools.go:20:	MaxResults *int64 `min:"1" type:"integer" required:"true"`
aws-sdk-go-v2/service/cognitoidentityprovider/api_op_ListUserImportJobs.go:19:	MaxResults *int64 `min:"1" type:"integer" required:"true"`
aws-sdk-go-v2/service/ec2/api_op_DescribeByoipCidrs.go:25:	MaxResults *int64 `min:"1" type:"integer" required:"true"`
```
❯ ag 'MaxResults \*int'  | grep -v 'required'
(most other services)

This param's definition does differ with other service's, so as the behavior. (e.g. leaving out MaxResults in ListUserPoolClientsInput would make the call refuses to return results, while leaving out MaxResults for other ListXXXXXXXInput would have a default value at aws's side). 

So I guess we have to go towards the paginator way to support this one fully, could be separate changes/PR in sharable files.

Reply:
This one is by trial-and-error when 

```
❯ ./terraformer import aws --resources appsync --regions ap-southeast-1
2020/03/30 01:12:08 aws importing region ap-southeast-1
2020/03/30 01:12:08 aws importing... appsync
2020/03/30 01:12:09 BadRequestException: maxResults must be greater than 0 and less than or equal to 25.
	status code: 400, request id: 522b8519-d992-4747-9a0d-78b8da318640
2020/03/30 01:12:13 aws Connecting.... 
2020/03/30 01:12:13 aws save appsync
2020/03/30 01:12:13 aws save tfstate for appsync
```
comparing `ListPipelinesRequest` and `ListGraphqlApisRequest` that paginator  might be better to be built into aws go sdk , I may raise an issue asking them there